Dockerfile.cockpit

```diff
@@ -5,5 +5,9 @@ RUN dnf install -y cockpit-ws cockpit-kubernetes && dnf clean all
 RUN echo $'[Negotiate]\nCommand = /usr/libexec/cockpit-stub\n[WebService]\nShell = /shell/simple.html' > /etc/cockpit/cockpit.conf
 RUN rm -rf /usr/share/cockpit/kubernetes/
 
+COPY welder-web*.rpm /tmp/
+RUN rpm -ih /tmp/welder-web*.rpm
```

Comment:
dnf install -y so we can pick up dependencies if later we have such

Reply:
In this case, there has to be with COPY instruction. Because for the ADD and COPY instructions, the contents of the file(s) in the image are examined and a checksum is calculated for each file. For the RUN instruction, cache checking will not look at the files in the container to determine a cache match.

Reply:
I'm not arguing about the COPY or RUN instructions. Instead of
`rpm -ih /tmp/welder-web*.rpm` (btw a better way is to rpm -Uh) use `dnf -y install /tmp/welder-web*.rpm` which will pull additional RPM requirements if/when the welder-web RPM package introduces them.

Reply:
Got it.